Gemfile

```diff
@@ -124,7 +124,8 @@ gem 'rack-x_served_by', '~> 0.1.1'
 gem 'rack-cors'
 gem 'roar-rails'
 
-gem 'reform', '~> 2.0.3', require: false
+gem 'reform', '~> 2.3.0', require: false
+gem 'reform-rails', '~> 0.2.2', require: false
```

Comment:
The reasoning for this upgrade is that ActiveRecord 6.1 support was added in `reform-rails` v0.2.2: https://github.com/trailblazer/reform-rails/blob/master/CHANGES.md#022

So, the split between `reform` and `reform-rails` happened even earlier.

Honestly, I think it worked even before the upgrade (at least the tests were passing), but I noticed the deprecation warnings (related to ActiveModel::Errors), and well, I thought an upgrade wouldn't hurt, although it required a non-trivial fix in `ThreeScale::Reform` module, which I am not sure is well done, see below.